src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/Completion/CompletionHandler.cs

```diff
@@ -99,10 +109,29 @@ private static CompletionItem CreateCompletionItem(XamlCompletionItem xamlComple
                     Title = CreateEventHandlerCommandTitle
                 };
             }
+            else if (xamlCompletion.RetriggerCompletion)
+            {
+                // Retriger completion after commit
+                item.Command = new Command()
+                {
+                    CommandIdentifier = StringConstants.RetriggerCompletionCommand,
+                    Title = RetriggerCompletionCommandTitle
+                };
```

Comment:
I don't think the title is needed here

Reply:
You have to set it otherwise the code will keep throwing. As the title is not viewed anywhere I just use a simple string instead of a localized string.

---
In reply to: [618731026](https://github.com/dotnet/roslyn/pull/52848#discussion_r618731026) [](ancestors = 618731026)

Reply:
OK, never mind.

---
In reply to: [618735737](https://github.com/dotnet/roslyn/pull/52848#discussion_r618735737) [](ancestors = 618735737,618731026)